content/03.prior-knowledge.md

```diff
@@ -0,0 +1,22 @@
+### Techniques that build on prior knowledge and indirectly related data are necessary for many rare disease applications {.page_break_before}
+
+#### Among ensemble learning, cascade learning is more successful in cases of rare disease
```

Comment:
this header I think could be made a little more concise and punchy, maybe some thing like:

```suggestion
#### Wisdom of the crowd: rare disease applications of ensemble methods
```